README.md

```diff
@@ -26,9 +27,13 @@ core-libraries-committee at haskell dot org
 
 ## `base` package
 
-The primary responsibility of CLC is maintenance of `base` package (owned by GHC Team).
+The primary responsibility of CLC is to manage API changes of `base` package. The ownership of `base` belongs to GHC developers, and they can maintain it freely without CLC involvement as long as changes are invisible to clients. Changes which affect performance or laziness and similar are deemed visible. Documentation changes normally fall under GHC developers purview, unless significant ones (e. g., adding or changing type class laws).
```

Comment:
"except significant ones", not "unless significant ones"

Reply:
Surely this is the place to link to PROPOSALS.md?  Something like:
```
Proposals to change the API of the base package are managed by the 
base-package proposals process, described here (link). 
```
I'd suggest moving the Quick Links at the top to here.  They have too little context at the top.